Tasks/NuGetCommandV2/Common/VstsNuGetPushToolUtilities.ts

```diff
@@ -1,16 +1,15 @@
 // Placed as a separate file for the purpose of unit testing
-import * as path from "path";
 import * as commandHelper from "nuget-task-common/CommandHelper";
+import * as path from "path";
 
 export function getBundledVstsNuGetPushLocation(): string {
-    const vstsNuGetPushPaths: string[] = ["VstsNuGetPush/0.13.0"];
+    const vstsNuGetPushPaths: string[] = ['VstsNuGetPush/0.17.0'];
```

Comment:
Should this array contain both versions?

Reply:
not very familiar with this, but my vote is to add 0.13.0 also only if it's just for safety, as a fallback, but so long as we know it will always try to find it in 0.17.0 first : )